dockerfiles/bases/tiflash-base/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# hub.pingcap.net/bases/tiflash-base:v1.5.0
+# hub.pingcap.net/bases/tiflash-base:v1.5
+# hub.pingcap.net/bases/tiflash-base:v1
+FROM hub.pingcap.net/bases/pingcap-base:v1.5.0
+RUN dnf install --allowerasing -y wget && dnf clean all
```

Comment:
should we also add curl in `tiflash-base` to make it simple and unified.

Reply:
they dont need it, the simpler, the better